sql-statements/sql-statement-select.md

```diff
@@ -47,10 +64,19 @@ SelectLockOpt ::=
 TableList ::=
     TableName ( ',' TableName )*
 
+WhereClause ::=
+    "WHERE" Expression
```

Comment:
Should the following 3 `Expression` also include both names/aliases from fields and placement ids (column number)?